src/main/kotlin/com/cognifide/gradle/aem/environment/hosts/HostsOptions.kt

```diff
@@ -0,0 +1,14 @@
+package com.cognifide.gradle.aem.environment.hosts
+
+class HostsOptions {
+    var list = listOf(
+            Host("127.0.0.1", "example.com"),
+            Host("127.0.0.1", "demo.example.com"),
```

Comment:
this cannot be hardcoded in GAP but could be a part of example / gradle-aem-multi